how-to/customize-workspace/client/src/framework/fdc3/1.2/mapper.ts

```diff
@@ -37,7 +37,7 @@ function getManifest(app: AppDefinition): unknown {
 }
 
 function getTags(app: AppDefinition & { tags?: string[] }): string[] {
-	const tags: string[] = app.tags ?? [];
+	const tags: string[] = app.tags ?? app.categories ?? [];
```

Comment:
Shouldn't we be consistent here, neither tags or categories are part of the 1.2 spec - which is why they are an optional extension in the getTags definition rather than in the FDC31.2 AppDefinition - do both in the AppDefinition or preferably just in the getTags call